src/basicfuns.jl

```diff
@@ -407,3 +407,17 @@ function _softmax!(r, x, dims)
     end
     return r
 end
+
+"""
+$(SIGNATURES)
+
+Compute the complementary log-log, `log(-log(x))`.
+"""
+cloglog(x) = log(-log1p(-x))
+
+"""
+$(SIGNATURES)
+
+Compute the complementary double exponential, `exp(-exp(x))`.
```

Comment:
Same here, it seems this should be

```suggestion
Compute the complementary double exponential, `1 - exp(-exp(x))`.
```